### ISSUE TITLE
Listen to the Docker and Kubernetes event streams

### ISSUE THREAD

**Issue by EranGabber** [google/cluster-insight#13]
This way the data will always be fresh and the processing will be scalable .

**Comment by ttripp** [comment]
Does this apply?  Is this what should be done?

http://solinea.com/blog/tapping-kubernetes-events

**Comment by vasbala** [comment]
Yes, listening to the K8S event stream will get rid of a bunch of the current polling code, which would be nice. One thing the current polling code does though is communicate with each of the Docker daemons on the Pods to get lower level container and process information - the K8S event stream won't give this info. Still, getting rid of some of the current polling code is a plus.
